src/main/java/FibonacciEven.java

```diff
@@ -0,0 +1,19 @@
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class FibonacciEven {
+
+
+    public static List<Integer> generateOnlyEvenFibonacciNumbersSum(int numberFromUser){
```

Comment:
nazwa parametru mogła by być nieco lepsza - to nie jest numer od użytkownika tylko bardziej jakiś limit